apps/DataAggregator/Dockerfile

```diff
@@ -16,7 +16,17 @@ RUN apt-get update -y \
 
 FROM mcr.microsoft.com/dotnet/sdk:7.0-bookworm-slim AS build
 WORKDIR /src
-COPY . .
+COPY sdk /src/sdk
```

Comment:
1. SDKs are not needed for Docker builds
2. I guess we could slightly improve readability by inlining multiple copy operations into single `COPY` call?

_(applies to all Dockerfiles)_